entrypoint

```diff
@@ -12,9 +12,11 @@ git config --global user.email "91080706+conda-forge-webservices[bot]@users.nore
 conda activate base
 conda config --set show_channel_urls True
 conda config --add channels conda-forge
-conda config --show-sources
-conda config --set always_yes yes
+conda config --remove channels defaults || true
 conda config --set channel_priority strict
+conda config --set always_yes yes
+
+conda config --show-sources
 
 conda update --all --yes
```

Comment:
Tangentially related: I think it would make much more sense if we rebuild the container image with a higher frequency and drop the `conda update --all` here.
Running the `conda update` for every workflow job is unnecessary costly overall and it would be much more economical (and would also make the action slightly more responsive) to do it centralized by rebuilding the image.
WDYT?

Reply:
The issue is that we need a full rebuild anytime we update smithy or the pinnings repo. Even a cron job won't quite do the trick.

Reply:
Is there a way to check how often we run the action on a daily basis org-wide?
Even a 30-minute-interval cron job could be more economical overall.
(A workflow-dispatch trigger from smithy/pinnings would be even better, of course -- but I don't know this would interact with conda-forge-pinning being a feedstock itself.)

Reply:
We might be able to do something via our webservices as well. I'll think on it.

Reply:
Automation updating the automation so the automation is more efficiently updated to be more efficient at automating updates. Sounds about right!? :dizzy_face: :laughing: 
Thanks for considering/thinking about it!